agent/docs/none-and-external-platform.md

```diff
@@ -1,21 +1,25 @@
-# None Platform Support
+# None and External Platform Support
 
 ## Configuration Variable
 
 To enable None platform for compact and HA clusters:
 ````
 export AGENT_PLATFORM_TYPE=none
 ````
+To enable External platform for compact and HA clusters:
```

Comment:
For none platform, the default was SNO so saying `To enable None platform for compact and HA clusters:` is right for none but external always support all the 3 cluster topologies so please change `To enable External platform for compact and HA clusters:` to `To enable External platform for SNO, compact and HA clusters:`